node/plugin/cmd/main.go

```diff
@@ -130,7 +132,27 @@ func pluginOps(ctx *cli.Context) {
 	}
 	log.Printf("Info: ethclient created for url: %s", config.ChainRpcUrl)
 
-	tx, err := eth.NewWriter(logger, client, config.OperatorStateRetrieverAddr, config.EigenDAServiceManagerAddr)
+	contractDirectory, err := directory.NewContractDirectory(context.TODO(), logger, client,
```

Comment:
This method now has mixed usage, with `context.TODO` here, but `context.Background` below. Any reason to have different strategies within the same method?